src/main/java/duke/task/TaskManager.java

```diff
@@ -0,0 +1,182 @@
+package duke.task;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class TaskManager {
+    static final int INDEX_OF_KEYWORD_OF_TASK = 0;
+    static final String SEPARATOR = " | ";
+
+    private ArrayList<Task> tasks = new ArrayList<>();
+
+    public void readInLine(String[] words) {
+        if (words[0].equals("T")) {
+            tasks.add(new Todo(words[2]));
+            if (words[1].equals("true")) {
+                tasks.get(Task.getNumberOfTasks() - 1).setDone(); //else leave isDone as false
+            }
+        } else if (words[0].equals("D")) {
+            tasks.add(new Deadline(words[2], words[3]));
+            if (words[1].equals("true")) {
+                tasks.get(Task.getNumberOfTasks() - 1).setDone(); //else leave isDone as false
```

Comment:
The code is kinda simple and self-explanatory. Hence, I feel the comment is rather redundant. You may consider removing it :)